org.sugarlabs.Words.json

```diff
@@ -74,6 +74,10 @@
                 {
                     "type": "patch",
                     "path": "words-info.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "words-webkitfix.patch"
```

Comment:
also, can you check with Ibiam if this is something we can upstream?

Reply:
@chimosky Should I create a PR to change webkit2 version from 4.0 to 4.1 in upstream?